### ISSUE TITLE
Karma has a huge effect in the smart filter

### ISSUE THREAD

**Issue by theodedeken** [ludumdare/ludumdare#1891]
I don't know how intentional this is, but the previous times I participated in Ludum Dare I never had this much trouble getting to 20 ratings.

My understanding was if I rate 20 games or more I should get at least 20 ratings back, but it seems like you also need pretty high karma to even get listed in the top with the smart filter.

I understand that you want to encourage people to give more feedback, but it feels like you expect us to give feedback on literally every game we play and rate.

### Some examples
![Schermafdruk van 2020-04-25 15-29-24](https://user-images.githubusercontent.com/11034269/80282502-2a79ae00-8712-11ea-9cab-108f388a6c97.png)
My own [game](https://ldjam.com/events/ludum-dare/46/morituri-combonaturi) currently has received 16 ratings, while I have been giving 32 ratings to other games. I have a karma of 12 which feels ok for me as I comment on about one third of all games I play (I tend to only comment when I actually have good feedback). This has given me a smart score of **-17**. So I am nowhere near the top when listing games via the smart filter.

![Schermafdruk van 2020-04-25 15-22-27](https://user-images.githubusercontent.com/11034269/80282499-28afea80-8712-11ea-91ff-a9ac604d9b25.png)
Another great example is https://ldjam.com/events/ludum-dare/46/feed-him. Which at the time of writing has rated **80** games and has received only **10** ratings in return. This game is the highest in the classic filter by a long shot, but has an even worse smart score than my game with **-25**! Reason: the user has 0 karma.

This leads me to think that karma has way too big of an influence on the smart filter score. I don't know the inner workings of the algorithm, but looking through the rules it says that high karma gives you bonuses. However it seems like low karma nets you a penalty as well (which is not mentioned anywhere) and this penalty is so great that it even negates all the bonus you get from rating 8 games to 1.

Sorry if this feels like a rant and I am open to hear about your thoughts about this. It just feels that the last few days of rating for me have been very stressful as I feel obligated now to just leave a comment on each game I rate, which obviously is decreasing the quality of my feedback.

**Comment by local-minimum** [comment]
I agree that it does seem problematic how the balance works now. As a reference, for me who tries to write something useful on every game I play it places me way above both of you and since I'm almost at 20, that doesn't seem entirely right
![Screenshot_20200425-215118__02](https://user-images.githubusercontent.com/8041100/80289524-2c0d9b00-873f-11ea-8c36-813be468b3fe.jpg)

On the positive side, we're at almost 50% of the games above 20 votes so I have high hopes that most will get their grades in the end

**Comment by local-minimum** [comment]
Another effect of the changes is that it now very much works as a delayed reward. I'm not sure if that is good or bad though.

Before if you played 10 games (unless you'd already gotten way more votes than games you'd played) you got some new votes and comments back within the hour.

Now I play a couple of games and nothing happen, then I go to bed and wake up with a batch of new comments. This, I assume is because my comments need to generate karma before I rank up in smart balance, and that takes time.

**Comment by theodedeken** [comment]
I think I tracked the change down to this commit: 5480f71d98eeb3622f50627d3d300f97d0681f53 by @mikekasprzak 

Here the smart algorithm is changed such that any reviews you give above a threshold of **5** are not counted towards the score anymore. Afterwards only karma and reviews received influence your score.

This seems really low and has the effect that users with low (or no) karma are unlikely to reach the top of the smart filter.

**Comment by local-minimum** [comment]
5 sounds really low, 20-30 seems more reasonable given the current algorithm.

**Comment by mikekasprzak** [comment]
I'm sitting on a change that gradually increases the cap by 1 every day, but I'm tempted to still cap it at 10 or 15.

I get flooded with complaints every event regarding people that rate games without actually playing them. We also get so many submissions now, that placing in the top 100 games is extremely rare.

Yes, this change is aggressive. Some folks will find it inconvenient. And it is going to take a few events to get the algorithm and messaging right. But this way, everybody that takes the event seriously gets something out of it.

That, and I'd rather folks be mad at an algorithm than each-other. Tattle-tale emails are getting reeeealy old.

**Comment by theodedeken** [comment]
Thanks for the clarification. I see your point now. 

I agree that the algorithm needs some balancing and it needs to be clearer on the site that karma is the main factor in determining your smart score.

**Comment by local-minimum** [comment]
I don't mind the changes either if that was the intention. It should probably have been communicated in blinking caps in the banner.

As for increasing the cap by one every day you'd end up with 5 + 21 or 21 if we start with 0 as max (which will be a little weird on Saturday and Monday. Perhaps start with 1-5 increase with one every second day then which will put the final at 10-15.  Even if we add 1 every day it's still possible to do a first (or more probably last day) voting spree with little care for actually testing the games quite similarly to the old 50 votes cap.

**Comment by local-minimum** [comment]
I have a question though. Right now I'm at:
```
Karma for Feedback given: 42
Ratings given: 24
Ratings received: 34.2857
Classic Balance: 90.9718 *
Smart Balance: -13.5458 *
```
And that almost places me on the first page. Now since cap is at 5 votes is that also my negative effects on vote received? Because if it is, then if I feel like I'm happy with my contribution and stop coming back to the page until end of voting. Not giving love to people who comment on my game for a week or two. Then will I not pretty much park myself on the first page, or close to it?
